### ISSUE TITLE
Q object does not build query properly

### ISSUE THREAD

**Issue by kalombos** [Infinidat/infi.clickhouse_orm#72]
```python
>>> qs = StorageUnitHistory.objects.filter(onlineprice=5)\
.filter(Q(brandname_id=4) | Q(brandname_id=5) | Q(brandname_id=6))
>>> qs.conditions_as_sql()
onlineprice = 5.0 AND ((brandname_id = 4) OR (brandname_id = 5)) OR (brandname_id = 6)
```
It is wrong because i need: 
```
onlineprice = 5.0 AND ((brandname_id = 4) OR (brandname_id = 5) OR (brandname_id = 6))
```

**Comment by ishirav** [comment]
Thank you for the bug report, I'll look into it.
As a workaround you could use: `filter(brandname_id__in=[4, 5, 6])`

**Comment by kalombos** [comment]
@ishirav thank you, i didn't think about this simple workaround and used this monkey patch:

```
from infi.clickhouse_orm.query import Q

def to_sql(self, model_cls):
    if self._fovs:
        sql = ' {} '.format(self._mode).join(fov.to_sql(model_cls) for fov in self._fovs)
    else:
        if self._l_child and self._r_child:
            sql = '({} {} {})'.format(
                    self._l_child.to_sql(
                        model_cls), self._mode, self._r_child.to_sql(model_cls)
            )
        else:
            return '1'
    if self._negate:
        sql = 'NOT (%s)' % sql
    return sql

# Monkey patch :(
Q.to_sql = to_sql
```

The changes only in this line **sql = '({} {} {})'.format(...** I removed a pair of brackets.

**Comment by ishirav** [comment]
Fixed in v1.0.3
